daprdocs/content/en/reference/components-reference/supported-bindings/kafka.md

```diff
@@ -78,6 +82,8 @@ spec:
 | `saslMechanism` | N | Input/Output | The SASL authentication mechanism you'd like to use. Only required if `authtype` is set to `"password"`. If not provided, defaults to `PLAINTEXT`, which could cause a break for some services, like Amazon Managed Service for Kafka. | `"SHA-512", "SHA-256", "PLAINTEXT"` |
 | `initialOffset`   | N | Input | The initial offset to use if no offset was previously committed. Should be "newest" or "oldest". Defaults to "newest". | `"oldest"` |
 | `maxMessageBytes` | N | Input/Output | The maximum size in bytes allowed for a single Kafka message. Defaults to 1024. | `"2048"` |
+| `heartbeatInterval` | N | Input | The interval between heartbeats to the consumer coordinator. The value should be set to at most a 1/3 of `sessionTimeout`'s value. Defaults to 3s. | `5s` |
+| `sessionTimeout`    | N | Input | The maximum time between heartbeats before the consumer is considered inactive and will timeout. Defaults to 10s. | `30s` |
```

Comment:
Do you really mean "maximum time between heartbeats" here? Shouldn't this say "Maximum session time before all heartbeats with the heartbeatInterval stop" or similar?
Just trying to read this from a user perspective and understand how to use this? And Does this directly map to a Kafka property?

Reply:
After considering this for a bit, I do agree that it's not exactly clear, so I changed to the following (taking part of Kafka's documentation):
"The timeout used to detect client failures when using Kafka’s group management facility. If the broker fails to receive any heartbeats from the consumer before the expiration of this session timeout, then the consumer will be removed and initiate a rebalance."

I could also take most of the explanation on the `sessionTimeout` from confluent's documentation entirely instead of trying to change it.